install.sh

```diff
@@ -604,6 +604,11 @@ install_werf_system_dependencies() {
     return 0
   fi
 
+  VERSION_ID=$(lsb_release -r | cut -f2)
```

Comment:
What if there is no `lsb_release`? Take a look at the `get_linux_distro()` function, that's how it should be done.